Cartfile

```diff
@@ -1,4 +1,3 @@
-github "Alamofire/Alamofire" ~> 4.6
-github "Thomvis/BrightFutures" ~> 6.0
-github "Quick/Nimble" ~> 7.0
-github "SwiftyJSON/SwiftyJSON" ~> 4.0
+github "Thomvis/BrightFutures" ~> 8.0
+github "Quick/Nimble" ~> 8.0
+github "SwiftyJSON/SwiftyJSON" ~> 5.0.0
```

Comment:
After having `Codable` available to us since Swift 4, I'd encourage you to avoid bringing in more dependencies for things that Swift can do for by just using what `Foundation` provides.

Reply:
Also something I agree on. Original the branch was done by @andrewspinks . When I first started working on this my goal was to get it to a point that it can be merged back. 
Since Codable came up I thought about this, but the top priority was to get everything else working right first.